client/src/routes/[anime_id]/+page.svelte

```diff
@@ -1,32 +1,85 @@
 <script lang="ts">
     import { enhance } from "$app/forms";
-    import Anime from "../anime.svelte";
+    import { toast } from "svelte-sonner";
     import type { PageData } from "./$types";
 
     export let data: PageData;
 </script>
 
-<Anime
-    title={data.anime.title}
-    mal_id={data.anime.mal_id}
-    image={data.anime.images.webp.image_url}
-/>
+<div class="grid grid-flow-col gap-8">
+    <div class="relative w-full max-w-sm overflow-hidden rounded-lg shadow-lg">
+        <img
+            src={data.anime.images.webp.large_image_url}
+            alt={data.anime.title}
+            class="h-auto w-full"
+        />
+        <div
+            class="absolute bottom-0 left-0 w-full bg-gradient-to-t from-black to-transparent p-4 text-white"
+        >
+            <h2 class="text-lg font-semibold shadow-lg">{data.anime.title}</h2>
+        </div>
+    </div>
+    <div>
+        <p class="text-base leading-relaxed text-gray-700">
+            {data.anime.synopsis}
+        </p>
+        <form
+            action="?/addToFavorites"
+            method="post"
+            use:enhance={() =>
+                async ({ result, update }) => {
+                    console.log(result);
+                    if (result.type === "success" && result.data?.["success"]) {
+                        toast.success("Favourite has been added successfully");
+                    }
+                    if (
+                        result.type === "success" &&
+                        !result.data?.["success"]
+                    ) {
+                        toast.error(`${result.data?.["message"]}`);
+                    }
+                    update();
+                }}
```

Comment:
It's better to watch for form data return from actions, cos this one OLNY works with js, and the proposed solution works without it, and you get the same result:

```
export let form: ActionData;

$: if (!form.success) {
toast.error(.....)
}